components/src/Tables/Table.js

```diff
@@ -25,6 +26,54 @@ export const Cell = styled.td`
 
 Cell.defaultProps = defaultProps;
 
+export const Button = styled(QuietButton)`
+  ${ ({ theme }) => css`
+    padding: ${theme.space.half}px;
+    border-width: 0;
+    border-radius: 50%;
+    transition: none;
+    color: ${theme.colour.neutral[600]};
+    ${IconWrapper}{
+      width: 1.5em;
+      height: 1.5em;
+    }
+    ${IconWrapper} > svg{
```

Comment:
Can we export and target the`Svg` component directly?

Reply:
As it is, this selector is overly-tied to internal knowledge of the Icon components markup.

Reply:
Can we extract this one from this pull request?

Reply:
I do not believe we can. I do not want to allow poor coding practices to pile up in our code base and accrue debt.